src/iris/bin/sync_targets.py

```diff
@@ -163,12 +172,18 @@ def sync_from_oncall(config, engine, purge_old_users=True):
         logger.warning('No users found. Bailing.')
         return
 
-    oncall_team_names = fetch_teams_from_oncall(oncall)
+    # get teams from oncall-api and separate the list of touples into two lists of name and ids
+    oncall_team_response = list(zip(*fetch_teams_from_oncall(oncall)))
+    oncall_team_names = oncall_team_response[0]
```

Comment:
If fetch_teams_from_oncall returns no result, the [0] and [1] here will raise IndexErrors